.github/workflows/ci-unix-static-sanitized-memory.yml

```diff
@@ -0,0 +1,119 @@
+# Memory sanitizer needs to be in its own file because it is slower than other sanitizers.
+name: CI Unix Static Sanitized Memory
+on:
+  push:
+  pull_request:
+    paths:
+      - ".github/actions/**"
+      - ".github/workflows/ci-unix-static-sanitized-memory.yml"
+
+permissions:
+  contents: read
+
+jobs:
+  build-static-sanitized-memory:
+    runs-on: ubuntu-latest
+
+    env:
+      CC: clang
+      CXX: clang++
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: ./.github/actions/setup-linux
+        if: runner.os == 'Linux'
+        id: setup_linux
+        with:
+          codec-aom: "LOCAL"
+          codec-dav1d: "LOCAL"
+          libyuv: "LOCAL"
+          extra-cache-key: memory
+      - uses: ./.github/actions/setup-macos
```

Comment:
Delete the setup-macos step? This workflow is only run on ubuntu-latest now (see line 15).

Reply:
Done